hc/api/tests/test_create_check.py

```diff
@@ -11,27 +11,27 @@ def setUp(self):
         super(CreateCheckTestCase, self).setUp()
 
     def post(self, data, expected_error=None):
-        r = self.client.post(self.URL, json.dumps(data),
+        response = self.client.post(self.URL, json.dumps(data),
                              content_type="application/json")
 
         if expected_error:
-            self.assertEqual(r.status_code, 400)
+            self.assertEqual(response.status_code, 400)
             ### Assert that the expected error is the response error
 
-        return r
+        return response
```

Comment:
I don't see any assertion here

Reply:
@ranchow This is the post method being defined in the setup for later use in test cases, Should it have any assert. kindly clarify

Reply:
@ranchow  This is predefined post method that we are using to perform posts in test cases below, should it have an assert? given that it will yield different responses depending on how it's called?